pavelib/utils/test/suites/bokchoy_suite.py

```diff
@@ -396,16 +397,31 @@ def cmd(self):
         """
         data_dir = os.path.join(self.report_dir, 'data')
         url = "https://raw.githubusercontent.com/edx/pa11ycrawler-ignore/master/ignore.yaml"
-        return [
-            "scrapy",
-            "crawl",
-            "edx",
-            "-a",
-            "port=8003",
-            "-a",
-            "course_key={key}".format(key=self.course_key),
-            "-a",
-            "pa11y_ignore_rules_url={url}".format(url=url),
-            "-a",
-            "data_dir={dir}".format(dir=data_dir)
-        ]
+        if self.single_url:
+            return [
+                "scrapy",
+                "crawl",
+                "edx",
+                "-a",
+                "course_key={key}".format(key=self.course_key),
+                "-a",
+                "pa11y_ignore_rules_url={url}".format(url=url),
+                "-a",
+                "data_dir={dir}".format(dir=data_dir),
+                "-a",
+                "single_url={url}".format(url=self.single_url),
+            ]
+        else:
+            return [
+                "scrapy",
+                "crawl",
+                "edx",
+                "-a",
+                "port=8003",
+                "-a",
+                "course_key={key}".format(key=self.course_key),
+                "-a",
+                "pa11y_ignore_rules_url={url}".format(url=url),
+                "-a",
+                "data_dir={dir}".format(dir=data_dir)
+            ]
```

Comment:
There's a lot of repeated code here. How about moving the list of common args into a variable up above the `if` statement? That way you can add case-specific args within the if statement and just return the variable below.